style.css

```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family:
+    'Poppins',
+    sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+}
+
+body {
+  max-width: 400px;
+  width: 100%;
+  min-height: 100vh;
+  background-color: #fff;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 44px;
+  padding: 44px 30px;
+  background-color: white;
+}
+
+.my_logo {
+  color: #6070ff;
+  width: 76px;
```

Comment:
[OPTIONAL]
- Kindly increase the width of `my-logo` to make your logo look better.
<img width="200" alt="image" src="https://user-images.githubusercontent.com/77190203/148655925-ff0e935f-97c2-4fbe-8541-7ae191f7fa16.png">

Reply:
I will do that.